app/views/admin/people/edit.html.erb

```diff
@@ -0,0 +1,4 @@
+<center><h1 class="droidFont">Editing Person</h1></center>
```

Comment:
I see other places you're using the droidFont class on p tags, too.  Why not just style the body maybe?